### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by KairaMiu**
- Create edgefarm yocto layer
- Create `edgefarm-image` which is derived from `ci4rail-base-image`
- Add documentation content

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,19 @@
...
+
+## Dependencies
+
+This layer depends on:
+
+* The layer itself has no dependencies right now.
+
+## Build
+
+See [build instructions](https://github.com/ci4rail/yocto-images/tree/cleanup#building) from Ci4Rail repository for build definitons and scripts for yocto builds.
```
Comment (KlausPopp) [inline README.md]: must be changed to main branch finally

File: README.md [hunk 2]
```diff
@@ -1,2 +1,19 @@
-# meta-edgefarm
-EdgeFarm Yocto Layer
+# meta-ci4rail-bsp
```
Comment (KlausPopp) [inline README.md]: Change to meta-edgefarm

File: README.md [hunk 3]
```diff
@@ -1,2 +1,19 @@
...
+
+Yocto Software layer for EdgeFarm Services.
+
+This layer also includes images prepared for the Ci4Rail supported HW devices.
```
Comment (KlausPopp) [inline README.md:5]: Is that true?

**Review by KlausPopp — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (KairaMiu) [reply README.md]: argf

File: README.md (hunk 1 above)
Comment (KairaMiu) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (KairaMiu) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (KairaMiu) [reply README.md:5]: Isn't it? 
Reference: `recipes-images/images/edgefarm-image.bb`

Comment (KairaMiu) [reply README.md:5]: maybe this sentence is just confusing?

**Review by KlausPopp — APPROVED** [review]

**Pull request merged**